demos/yolopv2/Dockerfile

```diff
@@ -12,6 +12,7 @@ COPY requirements.txt /demo/requirements.txt
 
 RUN pip install -r /demo/requirements.txt
 
-RUN pip install git+https://github.com/tryolabs/norfair.git@master#egg=norfair
+# Install PyTorch from URL
```

Comment:
This comment is not helpful.

Why are we installing this version of PyTorch?

Reply:
We had problems with the versions of Torch used. I didn't see that the actions failed. It was solved using the ones from the docker image.